app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  after_create :update_posts_counter
+  has_many :comments
+  has_many :likes
+  belongs_to :author, class_name: 'User'
+
+  def update_posts_counter
+    User.find_by(name: author.name).increment!(:posts_counter)
+  end
+
+  def latest_comments
+    comments.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] Good job so far 👏 👏. In addition to what you have done, I would suggest adding tests for the following custom method in the `Post` model that helps you query the latest comments on a post 😃 👍 . This is among the project requirements as you can see below 👇 
![image](https://user-images.githubusercontent.com/90222110/215150862-cff865e0-9365-4efa-ae55-eb4a54349a80.png)